src/Apps/Artist/Routes/Consign/Components/ArtistConsignMeta.tsx

```diff
@@ -1,33 +1,67 @@
 import React from "react"
 import { Link, Meta, Title } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
 import { getENV } from "Utils/getENV"
 
+import { ArtistConsignMeta_artist } from "__generated__/ArtistConsignMeta_artist.graphql"
+
 interface ArtistConsignMeta {
-  artistName: string
-  artistHref: string
-  imageURL: string
+  artist: ArtistConsignMeta_artist
 }
 
 export const ArtistConsignMeta: React.FC<ArtistConsignMeta> = props => {
-  const { artistName, artistHref, imageURL } = props
+  const {
+    artist: {
+      name,
+      href,
+      targetSupply: {
+        microfunnel: { artworks },
+      },
+    },
+  } = props
+
+  const imageURL = artworks[0].artwork.image.imageURL
```

Comment:
Another place for optional chaining?

Reply:
Any artist in "top 20" will have an image, unless there's a problem at the service level. We now have a proper [ErrorBoundary](https://github.com/artsy/reaction/blob/master/src/Components/ErrorBoundary.tsx) up the stack which will handle cases like this.